packages/react-inbox/src/reducer.ts

```diff
@@ -70,10 +70,13 @@ export default (state: InboxState = {}, action) => {
   }
 
   case "inbox/MARK_MESSAGE_READ": {
+    const unreadMessageCount = (state.unreadMessageCount ?? 1) - 1;
```

Comment:
Not sure I understand why its -1 but approving. Maybe we could get a `Math.max(unreadMessageCount, 0)`

Reply:
This is a good idea